modules/custom/social_graphql/src/TypedData/Definition/PageInfoDefinition.php

```diff
@@ -13,28 +13,26 @@ class PageInfoDefinition extends ComplexDataDefinitionBase {
   /**
    * {@inheritdoc}
    */
-  public function getPropertyDefinitions() {
-    if (!isset($this->propertyDefinitions)) {
-      $info = &$this->propertyDefinitions;
-
-      $info['hasNextPage'] = DataDefinition::create('boolean')
-        ->setRequired(TRUE)
-        ->setLabel("Has next page")
-        ->setDescription("Whether the result-set has a next page");
-
-      $info['hasPreviousPage'] = DataDefinition::create('boolean')
-        ->setRequired(TRUE)
-        ->setLabel("Has previous page")
-        ->setDescription("Whether the result-set has a previous page");
-
-      $info['startCursor'] = DataDefinition::create('string')
-        ->setLabel("Start cursor")
-        ->setDescription("The cursor of the first result in the result-set");
-
-      $info['endCursor'] = DataDefinition::create('string')
-        ->setLabel("End cursor")
-        ->setDescription("The cursor of the last result in the result-set");
-    }
+  public function getPropertyDefinitions(): array {
+    $info = &$this->propertyDefinitions;
+
+    $info['hasNextPage'] = DataDefinition::create('boolean')
+      ->setRequired(TRUE)
+      ->setLabel("Has next page")
+      ->setDescription("Whether the result-set has a next page");
+
+    $info['hasPreviousPage'] = DataDefinition::create('boolean')
+      ->setRequired(TRUE)
+      ->setLabel("Has previous page")
+      ->setDescription("Whether the result-set has a previous page");
+
+    $info['startCursor'] = DataDefinition::create('string')
+      ->setLabel("Start cursor")
+      ->setDescription("The cursor of the first result in the result-set");
+
+    $info['endCursor'] = DataDefinition::create('string')
+      ->setLabel("End cursor")
+      ->setDescription("The cursor of the last result in the result-set");
```

Comment:
What did removing the `!isset` achieve here? 🤔 Do we know why it was here in the first place? Looks like a small performance optimization?